napari/_vispy/layers/labels.py

```diff
@@ -63,11 +64,15 @@
 uniform sampler2D texture2D_keys;
 uniform sampler2D texture2D_values;
 uniform vec2 LUT_shape;
-uniform int color_count;
-
 
 vec4 sample_label_color(float t) {
-    if (($use_selection) && ($selection != t)) {
+    if ($use_selection) {
+        if ($selection == t) {
+            texture2D(
+            texture2D_values,
+            vec2(0.0, 0.0)
```

Comment:
missing a return or a var definition here? Not sure what the intention was.

Reply:
missing return